src/main/kotlin/com/thelastpickle/tlpstress/commands/Run.kt

```diff
@@ -145,6 +145,11 @@ class Run(val command: String) : IStressCommand {
     @Parameter(names = ["--no-schema"], description = "Skips schema creation")
     var noSchema: Boolean = false
 
+    @Parameter(names = ["--deleterate", "--deletes"], description = "Deletion Rate, 0-1.  Workloads may have their own defaults.  Default is dependent on workload.")
+    var deleteRate : Double? = null
+
+    @Parameter(names = ["--rangedeleterate", "--rangedeletes"], description = "Range Deletion Rate, 0-1.  Workloads may have their own defaults.  Default is dependent on workload.")
```

Comment:
I think we shouldn't complexify configuration at this stage as it's not really clear to me if we would benefit from this setting.
Some workload will naturally generate full partition deletes only because they have no clustering key, and others because we can't easily select a range for the deletion (BasicTimeSeries for example).
I'm in favor of just keeping `--deletes` and removing `--rangedeletes` unless we can find a real use to it (which makes sense for stress testing).

@rustyrazorblade, wdyt?

Reply:
Yeah, you're right.  Let's go with just `--deletes` (and `--delete` should also work)